lldb/source/Utility/Checksum.cpp

```diff
@@ -22,8 +22,8 @@ Checksum &Checksum::operator=(const Checksum &checksum) {
 }
 
 void Checksum::SetMD5(llvm::MD5::MD5Result md5) {
-  const constexpr size_t md5_length = 16;
-  std::uninitialized_copy_n(md5.begin(), md5_length, m_checksum.begin());
+  static_assert(sizeof(md5) == 16, "MD5 expected to be 16 bytes/128 bits");
+  llvm::uninitialized_copy(md5, m_checksum.begin());
```

Comment:
Couldn't it be just `m_checksum = md5`?

Reply:
yeah, I wasn't sure why it was using uninitialized_copy for POD types which should be safe to just copy directly. I'll change it

Reply:
Done.